docker/dev/config/powermon.yaml

```diff
@@ -10,22 +12,22 @@ device:
     protocol: PI30
 
 #Separate the schedule from the device definition
-scheduling:
-  loop: 10 #seconds
-  schedules:
-  - name: QPIGS_5_minutes
-    type: loop
-    loopCount: 6 #will run every 1 minute
-    commands:
-      - command: QPIGS
-        type: basic #default command type is basic
-        outputs:
-        - type: mqtt
-          topic: TEST_INVERTER
-          tag: QPIGS
-          format: 
-            type: htmltable
-            filter: ^ac|^pv
+schedules:
+- name: one_minute
+  type: loop
+  loop_count: 6 #will run every 1 minute
+
+commands:
+- name: QPIGS_every_minute
+  command_query: QPIGS #Must be in the protocol command list
```

Comment:
why did you go to command_query?

Reply:
the word command was really overused and I was getting them mixed up.

Reply:
well thats true - think thats why i went for name
query implies something compared to the setters tho